pika/connection.py

```diff
@@ -331,6 +331,12 @@ def heartbeat(self):
             connection tuning or callable which is invoked during connection tuning.
             None to accept broker's value. 0 turns heartbeat off. Defaults to
             `DEFAULT_HEARTBEAT_TIMEOUT`.
+
+        Note: The connection may still be timed out by the broker, regardless of the
+              heartbeat setting, if the client takes too long to accept new messages.
+              Setting `prefetch_count` to 1 on the channel is one way to prevent this
```

Comment:
@michaelklishin, do you agree?

Reply:
I'm afraid I don't. Client responsiveness is not tied to a particular prefetch setting and 1 is a very conservative value. Again, I primarily dislike the incredible generality of the language here.